README.md

```diff
@@ -9,21 +15,22 @@ In here you will find everything that runs inside Unreal Engine.
 This includes the world, all assets and the AirSim server.
 The AirSim server uses the AirLib shared code (see `/AirSim/AirLib`).
 
+`/Simulator` is the simulation control system. This is a ROS workspace.
```

Comment:
this is maybe outside of the scope of this PR, but somehow I'm not fully convinced about naming this part of our project the simulator. There is a simulator inside UE as well, which makes things a bit confusing. Besides, in this workspace we aren't simulating anything, we are simply doing message conversions and orchestrating the simulation at a high level

Reply:
I agree. Naming it the Simulator is confusing

Reply:
Oke! What name would fit better?

Reply:
something like simulation_bridge or simulation_interface..

Reply:
Conceptional I think the simulator is the control point of the whole simulation. The world is just a slave to whatever the simulator tells it what to do. The simulator will implement most competition business logic. Sure the world will do all phisics but not without instructions from the simulator. The name 'interface' is often used for components that expose some business logic hidden in another component externally. So i feel like this thing is a lot more than just an interface. I feel a bridge connects two systems directly without adding much logic. The simulator connects the referee, world, autonomous system and adds lots of logic.

Maybe Operator could be used as a name?

Reply:
yeah you're right. I like operator a lot more!